### ISSUE TITLE
Cannot scrape any account anymore

### ISSUE THREAD

**Issue by JosiMou** [instaloader/instaloader#291]
**Describe the bug**
For like a week I have not been able to scrape any data from any kind of Instagram account anymore.
**To Reproduce**

```python
USERNAME = "oysho_sport"        # profile to download from



L = Instaloader(user_agent=user_agent, save_metadata=True, compress_json=True, download_geotags=False,
                download_pictures=False, download_videos=False, download_video_thumbnails=False, download_comments=True)
#LC = InstaloaderContext(user_agent = user_agent)

profile = Profile.from_username(L.context, USERNAME)


# -------------------------------------------------------------------------------------------- #
X_percentage = 0.6    # percentage of posts that should be downloaded

profile = Profile.from_username(L.context, USERNAME)
posts_sorted_by_likes = sorted(profile.get_posts(), key = lambda p: p.likes + p.comments)

def scrapeInstaTOPx():
    try:
        for post in islice(posts_sorted_by_likes, ceil(profile.mediacount * X_percentage / 100)):
            L.download_post(post, USERNAME)
    except ValueError:
        pass
```

**Expected behavior**
Whenever I try to scrape data I receive the error message below.

**Error messages and tracebacks**
```
raceback (most recent call last):
  File "/Users/josefmoucachen/Documents/GitHub/RedDotPRJ_1/imageAnalyzer.py", line 140, in <module>
    scrapeInstaTOPx()
  File "/
~/InstagramScraper/instascraper.py", line 81, in scrapeInstaTOPx
    L.download_post(post, USERNAME)
  File "~//.virtualenvs/ThesisObjectDetection/lib/python3.6/site-packages/instaloader/instaloader.py", line 466, in download_post
    self.update_comments(filename=filename, post=post)
  File "~//.virtualenvs/ThesisObjectDetection/lib/python3.6/site-packages/instaloader/instaloader.py", line 254, in update_comments
    comments.extend(_postcomment_asdict(comment) for comment in post.get_comments())
  File "/Users/josefmoucachen/.virtualenvs/ThesisObjectDetection/lib/python3.6/site-packages/instaloader/instaloader.py", line 254, in <genexpr>
    comments.extend(_postcomment_asdict(comment) for comment in post.get_comments())
  File "/Users/josefmoucachen/.virtualenvs/ThesisObjectDetection/lib/python3.6/site-packages/instaloader/structures.py", line 304, in get_comments
    if self.comments == len(comment_edges):
UnboundLocalError: local variable 'comment_edges' referenced before assignment
```

**Instaloader version**
`instaloader --version` output.

**Additional context**

**Comment by aandergr** [comment]
> **Instaloader version**
>
> `instaloader --version` output.

Instaloader Version?

**Comment by JosiMou** [comment]
> > **Instaloader version**
> > `instaloader --version` output.
> 
> Instaloader Version?

4.2.3

**Comment by aandergr** [comment]
This has been reported with #272 and fixed with version 4.2.4. Upgrade Instaloader to its current version and reopen this issue if that doesn't fix this error.
